File: app/src/main/java/com/example/myapplication/Helpers/AppPackage.java
package com.example.myapplication.Helpers;

import android.content.pm.PackageManager;

public class AppPackage {

    /**
     * Checks if application is installed on device
     * */

    public static boolean isPackageInstalled(String packageName, PackageManager packageManager) {

        boolean found = true;

        try {

            packageManager.getPackageInfo(packageName, 0);
        } catch (PackageManager.NameNotFoundException e) {

            found = false;
        }

        return found;
    }
}

File: app/src/main/java/com/example/myapplication/helpers/AppPackage.kt
package com.example.myapplication.helpers

import android.content.pm.PackageManager

object AppPackage {
    /**
     * Checks if application is installed on device
     */
    fun isPackageInstalled(packageName: String?, packageManager: PackageManager): Boolean {
        var found = true
        try {
            packageManager.getPackageInfo(packageName, 0)
        } catch (e: PackageManager.NameNotFoundException) {
            found = false
        }
        return found
    }
}